lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -363,7 +363,7 @@
 
   &:hover,
   &:focus {
-    color: $link-color;
+    color: $forum-color-active-text;
```

Comment:
Is this different from our now accessible base blue? Is there any reason why you don't use `palette(primary, base)` as it should work on both white and our light gray backgrounds?